src/index.js

```diff
@@ -32,7 +32,7 @@ function init( {
 		root.querySelectorAll( selector ),
 		node => {
 			if ( isTouch ) {
-				node.addEventListener( 'touchstart', showPopup )
+				node.addEventListener( 'touchend', showPopup )
 			} else {
 				node.addEventListener( 'mouseenter', showPopup )
 			}
```

Comment:
btw, I found the below Wikipedia Hyperlinks doesn't go into the same logic, instead it handles only mouse event, can you also update the code for Wikipedia Hyperlinks?

```
if ( isTouch ) {
    // do touch event
} else {
    // do mouse event
}
```

Reply:
Good call, Wikipedia hyperlinks touch interactions are working for me now in Firefox and Chorme after last commit